source/frontend/src/features/mapSideBar/property/tabs/propertyResearch/update/models.ts

```diff
@@ -41,65 +44,64 @@ export class UpdatePropertyFormModel {
   public documentReference?: string;
   public researchSummary?: string;
   public propertyId?: number;
-  public researchFileId?: number;
-  public researchFileVersion?: number;
+
   public purposeTypes?: PropertyResearchFilePurposeFormModel[];
   public rowVersion?: number;
 
-  public static fromApi(base: Api_ResearchFileProperty): UpdatePropertyFormModel {
+  public researchFile: ApiGen_Concepts_ResearchFile | null = null;
+
+  public static fromApi(base: ApiGen_Concepts_ResearchFileProperty): UpdatePropertyFormModel {
     var model = new UpdatePropertyFormModel();
     model.id = base.id;
-    model.propertyName = base.propertyName;
-    model.displayOrder = base.displayOrder;
-    model.isLegalOpinionRequired =
-      base.isLegalOpinionRequired === undefined
-        ? 'unknown'
-        : base.isLegalOpinionRequired
-        ? 'yes'
-        : 'no';
-    model.isLegalOpinionObtained =
-      base.isLegalOpinionObtained === undefined
-        ? 'unknown'
-        : base.isLegalOpinionObtained
-        ? 'yes'
-        : 'no';
-    model.documentReference = base.documentReference;
-    model.researchSummary = base.researchSummary;
+    model.propertyName = base.propertyName ?? undefined;
+    model.displayOrder = base.displayOrder ?? undefined;
+    model.isLegalOpinionRequired = !exists(base.isLegalOpinionRequired)
+      ? 'unknown'
+      : base.isLegalOpinionRequired
+      ? 'yes'
+      : 'no';
+    model.isLegalOpinionObtained = !exists(base.isLegalOpinionObtained)
+      ? 'unknown'
+      : base.isLegalOpinionObtained
+      ? 'yes'
+      : 'no';
+    model.documentReference = base.documentReference ?? undefined;
+    model.researchSummary = base.researchSummary ?? undefined;
     model.propertyId = base.property?.id;
-    model.researchFileId = base.file?.id;
-    model.researchFileVersion = base.file?.rowVersion;
```

Comment:
lost this, not sure if that was intentional.

Reply:
this was only used to store the current research file. the object file does contain all that information so the individual members are not needed